src/lib/tzcode/CMakeLists.txt

```diff
@@ -3,36 +3,40 @@ add_library(tzcode STATIC
 target_link_libraries(tzcode)
 
 set(TZCODE_PROJECT tzcode-iana)
-set(TZCODE_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR})
-set(TZCODE_PREFIX_DIR tz)
-set(TZCODE_PREFIX ${TZCODE_BUILD_DIR}/${TZCODE_PREFIX_DIR})
+set(TZCODE_BINARY_DIR ${PROJECT_BINARY_DIR}/build/${TZCODE_PROJECT}/work)
+set(TZCODE_SOURCE_DIR ${PROJECT_BINARY_DIR}/build/${TZCODE_PROJECT}/src)
+
 set(TZDATA_TARGET main.zi)
+set(TZDATA_FULL_TARGET ${TZCODE_SOURCE_DIR}/${TZDATA_TARGET})
+
 set(TZCODE_SRC ${PROJECT_SOURCE_DIR}/src/lib/tzcode)
 set(TZLUA_SRC ${PROJECT_SOURCE_DIR}/src/lua)
 
-set(TZDATA_FULL_TARGET
-    ${TZCODE_PREFIX}/src/${TZCODE_PROJECT}/${TZDATA_TARGET})
-
 ExternalProject_Add(${TZCODE_PROJECT}
     GIT_REPOSITORY https://github.com/tarantool/tz
```

Comment:
Why not using as the third_party submodule?
When using an external module, we will not be able to get a self-contained tarball

Reply:
Ok, this target is custom, so it is ok. But this target explicitly requires a git binary even if not used

Reply:
Changed to use submodule `third_party/tz` instead of external project. Thanks!